docs/components/plugins/PaddedAPIBox.tsx

```diff
@@ -0,0 +1,13 @@
+import { PropsWithChildren } from 'react';
+
+import { APIBox } from '~/components/plugins/APIBox';
+
+type PaddedAPIBoxProps = PropsWithChildren<unknown>;
+
+export function PaddedAPIBox({ children, ...props }: PaddedAPIBoxProps) {
```

Comment:
```suggestion
export function PaddedAPIBox({ children, ...props }: APIBoxProps) {
```

Let's export `APIBoxProps` and re-use it:
https://github.com/expo/expo/blob/b3aeeeac3fc5d19bd716b6fe94d16984383c2c09/docs/components/plugins/APIBox.tsx#L14-L19